### PR TITLE
Fix docstring for nn.Softplus

### PR DESCRIPTION
**Pull request by lithuak**
Fixes nn.Softplus' docstring problem reported at #70498.

### CODE REVIEW

File: torch/nn/modules/activation.py [hunk 1]
```diff
@@ -776,10 +776,8 @@ def forward(self, input: Tensor) -> Tensor:
 
 
 class Softplus(Module):
-    r"""Applies the element-wise function:
-
-    .. math::
-        \text{Softplus}(x) = \frac{1}{\beta} * \log(1 + \exp(\beta * x))
+    r"""Applies the Softplus function :math:`\text{Softplus}(x) = \frac{1}{\beta} *
+    \log(1 + \exp(\beta * x))`, element-wise.
```
Comment (albanD) [inline torch/nn/modules/activation.py:780]: nit do you really need the `,` here?

Comment (lithuak) [reply torch/nn/modules/activation.py]: @albanD donno ) fixed

**Review by albanD — APPROVED** [review]
Thanks!

**Review by jbschlosser — APPROVED** [review]
LGTM! Thanks for the update :)

**Pull request closed without merging**